packages/fields/package.json

```diff
@@ -36,6 +36,7 @@
     "@keystonejs/adapter-knex": "^11.0.1",
     "@keystonejs/adapter-mongoose": "^9.0.1",
     "@keystonejs/app-admin-ui": "^7.3.0",
+    "@keystonejs/server-side-graphql-client": "^1.1.0",
```

Comment:
IMO the fields package should have minimum level of dependency, this can be achieved using context as done in earlier.

Reply:
The `server-side-grpahql-client` is an extremely lightweight package, especially when compared to the other dependencies of `fields`, so adding this dependency has a negligible impact.